devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy

```diff
@@ -172,6 +173,7 @@ private parseAttrMessage(description) {
 			log.debug "FAN MODE"
 			map.name = "thermostatFanMode"
 			map.value = FAN_MODE_MAP[it.value]
+			map.data = [supportedThermostatModes: state.supportedFanModes]
```

Comment:
Ah. This should say `supportedThermostatFanModes`.

Reply:
Oh jeez...